### ISSUE TITLE
Plugins - Incorrect List Count Announced [JAWS]

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1138]
"Description: On the Plugins page, when navigating with an active JAWS Screen reader, the list count announced does not match the actual number of items.
For example: There are 2 visible list items, but the screen reader [JAWS] announces “List with 3 items to activate press enter”

Note: This occurs intermittently with NVDA.

Expected Results: The screen reader [JAWS] should announce the correct number of items in the list (e.g., “List with 2 items” when there are 2 items).

Actual Results: Screen reader [JAWS] announces an incorrect list count (3 items instead of 2)."

1.3.1 Info and Relationships; Level A; WCAG 2.0
